cmd/signing_ethereum.go

```diff
@@ -0,0 +1,225 @@
+package cmd
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/keep-network/keep-common/pkg/chain/ethereum/ethutil"
+	"github.com/keep-network/keep-ecdsa/internal/config"
+	"github.com/urfave/cli"
+)
+
+// EthereumSigningCommand contains the definition of the `signing ethereum`
+// command-line subcommand and its own subcommands.
+var EthereumSigningCommand = cli.Command{
+	Name:  "ethereum",
+	Usage: "Ethereum signatures calculation",
+	Subcommands: []cli.Command{
+		{
+			Name:        "sign",
+			Usage:       "Sign a message using the operator's key",
+			Description: ethereumSignDescription,
+			Action:      EthereumSign,
+			ArgsUsage:   "[message]",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name: "eth-key-file,k",
+					Usage: "Path to the ethereum key file. " +
+						"If not provided read the path from a config file.",
+				},
+				cli.StringFlag{
+					Name:  "output-file,o",
+					Usage: "Output file for the signature",
+				},
+			},
+		},
+		{
+			Name:        "verify",
+			Usage:       "Verifies a signature",
+			Description: ethereumVerifyDescription,
+			Action:      EthereumVerify,
+			ArgsUsage:   "[ethereum-signature]",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "input-file,i",
+					Usage: "Input file with the signature",
+				},
+			},
+		},
+	},
+}
+
+const ethereumSignDescription = `Calculates an ethereum signature for a given message.
```

Comment:
Do you think we can add tabs in each line so it would look like this:
```
USAGE:
   keep-ecdsa signing ethereum sign [command options] [message]

DESCRIPTION:
   
  Calculates an ethereum signature for a given message.
  The message is expected to be provided as a string, it is later hashed with SHA-256
  and passed to Ethereum ECDSA signing. Signature is calculated in Ethereum specific
  format as a hexadecimal string representation of 65-byte {R, S, V} parameters.

  It requires an Ethereum key to be provided in an encrypted file. A path to the key file
  can be configured in a config file or specified directly with an 'eth-key-file' flag.

  The key file is expected to be encrypted with a password provided as KEEP_ETHEREUM_PASSWORD
  environment variable.
  
  The result is outputted in a common Ethereum signature format:
  {
    "address": "<address>",
    "msg": "<content>",
    "sig": "<signature>",
    "version": "2"
  }

  If 'output-file' flag is set the result will be stored in a specified file path.
```

Reply:
I'm not sure about it. We defined a very clean description as a string and how to print it is up to the library we use 🤷

Reply:
Up to you @nkuba, but ex. if you `man touch help` you would see a desc with indenations:
```
NAME
     touch -- change file access and modification times

DESCRIPTION
     The touch utility sets the modification and access times of files.  If any file does not exist, it is created with default permissions.

     By default, touch changes both modification and access times.  The -a and -m flags may be used to select the access time or the modification time individually.  Selecting both
     is equivalent to the default.  By default, the timestamps are set to the current time.  The -t flag explicitly specifies a different time, and the -r flag specifies to set the
     times those of the specified file.  The -A flag adjusts the values by a specified amount.

     The following options are available:......
```

I think in our case it should be enough to `tab` every line in the Description section to achieve the same layout.

Reply:
If I tab the description there is a mismatch between the first line (3 spaces) and subsequent lines (2 spaces).
```
DESCRIPTION:
   Calculates an ethereum signature for a given message.
  The message is expected to be provided as a string, it is later hashed with SHA-256
  and passed to Ethereum ECDSA signing. Signature is calculated in Ethereum specific
  format as a hexadecimal string representation of 65-byte {R, S, V} parameters.
```

We could overcome the problem and align the lines eventually but I think it would overcomplicate the code. I'm voting for simplicity and leaving it as is.

Reply:
Hm, right, there would need to be an empty line between the header and the description to align everything. Sure, let's leave it.